sagemathinc/cocalc
# site-settings: moving stripe to all the "extras" and refactor types

# Description

This moves the stripe settings into the general site-settings framework. It also refactors the types and adds a simple "header" to the settings, such that adjacent entries do not run together – which is confusing. In any case, the end result looks like that:

![Screenshot from 2020-03-04 15-07-16](https://user-images.githubusercontent.com/207405/75887437-1314f780-5e2a-11ea-9c2e-881c125ff4af.png)


# Testing Steps
1. I've not really tested it, besides that I came so far to make this screenshot and the DB values show up. So, I think this is low risk, overall, and easy to confirm that it works in "test"…
2. I also tested saving – there is a little check in `store` to skip such "header" entries.

# Relevant Issues

### [Checklist](https://github.com/sagemathinc/cocalc/wiki/PR-Checklist):

Front end:
